skimage/restoration/tests/test_inpaint.py

```diff
@@ -6,14 +6,16 @@
 from skimage._shared.testing import assert_allclose
 
 
-def test_inpaint_biharmonic_2d():
-    img = np.tile(np.square(np.linspace(0, 1, 5)), (5, 1))
+@testing.parametrize('dtype', [np.float32, np.float64])
```

Comment:
same thought here, why not test all relevant dtypes (all floats, or all types applicable to `inpaint`)